README.MD

```diff
@@ -95,9 +95,10 @@ You can mix both env variables & the device file
 
 ## data reporting
 By default the application reports some data back for usage statistics.
-This is to follow the amount  installs and which versions users are running, ...
-sm2 will always report what data its sending back.
-example of the data
+This is to follow the amount installs and which versions users are running, ...
```

Comment:
@nware-lab Sorry, but 
> This is to follow the amount installs and which versions users are running

seems like it's missing a word. Should it be:
> This is to follow the amount of installs and which versions users are running

?

Perhaps
> This is to report the number of Syncthing installations and their versions

would be better?